packages/shared/pkg/grpc/orchestrator-info/info.pb.go

```diff
@@ -26,22 +26,22 @@ const (
 type ServiceInfoStatus int32
 
 const (
-	ServiceInfoStatus_OrchestratorHealthy   ServiceInfoStatus = 0
-	ServiceInfoStatus_OrchestratorDraining  ServiceInfoStatus = 1
-	ServiceInfoStatus_OrchestratorUnhealthy ServiceInfoStatus = 2
+	ServiceInfoStatus_Healthy   ServiceInfoStatus = 0
+	ServiceInfoStatus_Draining  ServiceInfoStatus = 1
+	ServiceInfoStatus_Unhealthy ServiceInfoStatus = 2
 )
 
 // Enum value maps for ServiceInfoStatus.
 var (
 	ServiceInfoStatus_name = map[int32]string{
-		0: "OrchestratorHealthy",
-		1: "OrchestratorDraining",
-		2: "OrchestratorUnhealthy",
+		0: "Healthy",
+		1: "Draining",
+		2: "Unhealthy",
 	}
 	ServiceInfoStatus_value = map[string]int32{
-		"OrchestratorHealthy":   0,
-		"OrchestratorDraining":  1,
-		"OrchestratorUnhealthy": 2,
+		"Healthy":   0,
+		"Draining":  1,
+		"Unhealthy": 2,
```

Comment:
These maps are not used because we are using a binary backend for the gRPC protocol, so just enum value numbers are sent, and they are still the same.